<launch>
    <!--node pkg="tf2_ros" type="static_transform_publisher" name="world_broadcaster" args="0 0 0 0 0 0 1 world map" /-->
    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se" clear_params="true">
    <rosparam command="load" file="$(find mur_control)/config/ekf_template.yaml" />
    <remap from="odometry/filtered" to="mur/odom_filtered"/>
    </node>
    <!--node pkg="tf2_ros" type="static_transform_publisher" name="world_broadcaster" args="0 0 0 0 0 0 1 world map" /-->
</launch>
